### PR TITLE
Task/navigation start

### LINKED ISSUE #27: [US-271] Add link in navigation to voting
## Value Statement

As a user I want to be able to get to the voting by clicking the corresponding button in the navigation. 

## Description

userStory002

## Acceptance Criteria

- Add a Link to voting in the navigation
- After clicking the link the voting page will be displayed

## Subtasks

- [ ] Create feature branch
- [ ] Add Link to navigation
- [ ] Write component tests

## Complexity

Medium

### PR DESCRIPTION
**Pull request by StefanBehring**
## Add
- react router dom
- ButtonVoting component, spec, storybook
- ButtonHome Component, spec, storybook
- Navigation Component, spec, storybook
- Router to App
- Router to Storybook

## Edit
- Footer Component, spec

## Remove
- Voting spec (temporarily)

## Issues
- resolves #27

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+              <Voting />
+            </Route>
+            <Route path="/">
+              <Home />
+            </Route>
+          </Switch>
+        </Main>
+        <Footer />
+      </Wrapper>
+    </Router>
```
Comment (mbosselmann) [inline client/src/App.js:26]: 👍

File: client/src/components/Navigation/Buttons/ButtonHome/ButtonHome.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 50;
+  width: 80;
+`
+
+export default ButtonHome
```
Comment (mbosselmann) [inline client/src/components/Navigation/Buttons/ButtonHome/ButtonHome.js:25]: I have a suggestion according to your structure of files: Won't it be better to have one button folder at the same level as the other components where you can save all buttons you need in your app? Therefore you won't need another button folder for the other components and the hierarchical structure would be flatter. 🤔

File: client/src/components/Navigation/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <Nav>
+      <NavLink to="/">
+        <ButtonHome />
+      </NavLink>
+      <NavLink to="/voting">
+        <ButtonVoting />
+      </NavLink>
+    </Nav>
+  )
```
Comment (mbosselmann) [inline client/src/components/Navigation/Navigation.js:16]: Your usage of NavLinks and Buttons is not semantic. As far as I know it is bad practice to put a block element in an inline element. But you could also use the react router NavLink that you could style as flexbox. We can speak about this if you like. :-)

**Review by mbosselmann — COMMENTED** [review]
Well done! I have just two remarks and I would suggest that you think about my last remark. Feel free to contact me if my explanation isn't understandable. :-)

File: client/src/components/Navigation/Navigation.js (hunk 3 above)
Comment (StefanBehring) [reply client/src/components/Navigation/Navigation.js:16]: Changed it, thank you!

**Pull request merged**